### PR TITLE
Document the new DLNA DMS integration

### PR DESCRIPTION
**Pull request by chishm**
## Proposed change

Add documentation for a new integration, `dlna_dms`.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/66437
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3158
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/dlna_dms.markdown [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+ha_config_flow: true
+ha_codeowners:
+  - '@chishm'
+ha_domain: dlna_dms
+ha_platforms:
+  - media_source
+ha_ssdp: true
+---
+
+The `dlna_dms` integration allows you to browse and play media from a [DLNA Digital Media Server](https://www.dlna.org/). Configured DMS devices act as a [Media Source](/integrations/media_source/) and can be browsed in the Media panel.
```
Comment (frenck) [inline source/_integrations/dlna_dms.markdown]:
```suggestion
The DLNA Digital Media Server integration allows you to browse and play media from a [DLNA Digital Media Server](https://www.dlna.org/). Configured DMS devices act as a [Media Source](/integrations/media_source/) and can be browsed in the Media panel.
```

File: source/_integrations/dlna_dms.markdown [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+## Renaming
+
+The name/title of the DMS device is the same as the title of the config entry. It can be changed on the Integrations Configuration page from the three-dot menu.
+
+## Media source URIs
+
+Media source URIs for DLNA DMS look like `media-source://dlna_dms/<source_id>/<media_identifier>`.
+
+Here `<source_id>` is the a slugified name of the DMS device. E.g. "DLNA Server" becomes "dlna_server". If multiple DMS devices have the same name, an underscore and unique number will be appended to the end of some of them, e.g. "server", "server_1", "server_2".
```
Comment (frenck) [inline source/_integrations/dlna_dms.markdown]:
```suggestion
Here `<source_id>` is the slugified name of the DMS device. For example, "DLNA Server" becomes "dlna_server". If multiple DMS devices have the same name, an underscore and a unique number will be appended to the end of some of them, e.g., "server", "server_1", "server_2".
```

File: source/_integrations/dlna_dms.markdown [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+The `<media_identifier>` can have one of three forms:
+* `path/to/file` or `/path/to/file`: Slash-separated path through the Content Directory. This must refer to a unique media item.
+* `:ObjectID`: Colon followed by a server-assigned ID for an object.
+* `?query`: Question mark followed by a query string to search for, see [DLNA ContentDirectory SearchCriteria](http://www.upnp.org/specs/av/UPnP-av-ContentDirectory-v1-Service.pdf) for the syntax. The first result found will be used.
```
Comment (frenck) [inline source/_integrations/dlna_dms.markdown]:
```suggestion
The `<media_identifier>` can have one of three forms:

- `path/to/file` or `/path/to/file`: Slash-separated path through the Content Directory. This must refer to a unique media item.
- `:ObjectID`: Colon followed by a server-assigned ID for an object.
- `?query`: Question mark followed by a query string to search for, see [DLNA ContentDirectory SearchCriteria](http://www.upnp.org/specs/av/UPnP-av-ContentDirectory-v1-Service.pdf) for the syntax. The first result found will be used.
```

File: source/_integrations/dlna_dms.markdown [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+URIs generated while browsing will look like the Object ID form above. However, all three forms will work with the [media_player.play_media service](integrations/media_player/#service-media_playerplay_media).
```
Comment (frenck) [inline source/_integrations/dlna_dms.markdown]:
```suggestion
URIs generated while browsing will look like the Object ID form above. However, all three forms will work with the [media_player.play_media service](integrations/media_player/#service-media_playerplay_media).
```

**Review by frenck — APPROVED** [review]
Thanks, @chishm 👍

**Comment by chishm** [comment]
Thank you for the edits and merge :smile:

**Comment by frenck** [comment]
No problem! Thanks for contributing ❤️

**Pull request merged**